Src/cli/cli.cpp

```diff
@@ -43,449 +47,356 @@ bool checkNumber(std::string &number) {
 }
 
 void CLI::renderScreen() const {
-    system("clear");
+    int activeOptionIndex = currentScreen->getActiveOption();
+    clear();
 
-    printColored(BOLDWHITE, currentScreen->screenName.toStdString());
+    printColored(WHITE, currentScreen->screenName.toStdString(), true, true);
 
     for (const auto& notification: currentScreen->notifications) {
         printColored(WHITE, notification.toStdString());
     }
 
-    for (auto const &option: currentScreen->screenOptions) {
-        if (option.contains("enter")) {
-            std::string optionString = option.split("enter")[0].toStdString();
-            printColored(WHITE, optionString, false);
-            printColored(GREEN, "enter", false);
-            printColored(WHITE, ")");
-        } else {
-            printColored(WHITE, option.toStdString());
-        }
+    for (size_t i = 0; i < currentScreen->screenOptions.size(); ++i) {
+        QString option = currentScreen->screenOptions[i].optionName;
+        printColored(WHITE, std::to_string(i) + ". ", false);
+
+        int color = (i == static_cast<size_t>(activeOptionIndex)) ? FILLED_WHITE : WHITE;
+        printColored(color, option.toStdString());
     }
 }
 
-void CLI::changeScreen(std::shared_ptr<Screen> screen) {
-    currentScreen = std::move(screen);
+void CLI::changeScreen(const QString &screenName) {
+    qDebug() << "Changing screen to " << screenName;
```

Comment:
remove qdebug pleaseeee

Reply:
Fixed in 3812eeba4db3f395c3738a1ce1a7c82a5aa25d54